### PR TITLE
Added index in INSTALL.md file

### PR DESCRIPTION
**Pull request by tejasc0**
I added index in install.md file. Index is important to navigate in Installation page. For nested list in index, i had to remove the unnecessary links of plugin managers.

### CODE REVIEW

**Comment by danielshahaf** [comment]
> I added index in install.md file. Index is important to navigate in Installation page.

Sounds good to me.

> For nested list in index, i had to remove the unnecessary links of plugin managers.

I don't think "unnecessary" is the word you were looking for.  It's rather an undiplomatic way to disagree with whoever added those links.

In any case, I don't think the links are "unnecessary".  They do add value.  That doesn't mean they _must_ stay, but it does mean that we should look for alternatives that preserve them.  Is there any other way to add anchors in Markdown in general, or in GitHub's version of Markdown in particular?  Or to add an automatically-generated table of contents?

**Comment by tejasc0** [comment]
> > I added index in install.md file. Index is important to navigate in Installation page.
> 
> Sounds good to me.
> 
> > For nested list in index, i had to remove the unnecessary links of plugin managers.
> 
> I don't think "unnecessary" is the word you were looking for. It's rather an undiplomatic way to disagree with whoever added those links.
> 
> In any case, I don't think the links are "unnecessary". They do add value. That doesn't mean they _must_ stay, but it does mean that we should look for alternatives that preserve them. Is there any other way to add anchors in Markdown in general, or in GitHub's version of Markdown in particular? Or to add an automatically-generated table of contents?

I added anchor links and also mapped all links to Index, now everything works fine.

File: INSTALL.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
 How to install
 --------------
 
+* [Using packages](#Using-packages)
+* [In your ~/.zshrc](#In-your-~/.zshrc)
+* [Using packages](#Using-packages)
+* [With a plugin manager](#With-a-plugin-manager)
+  - [Antigen](#Antigen)
+  - [Oh-my-zsh](#oh-my-zsh)
+  - [Prezo](#Prezto)
```
Comment (danielshahaf) [inline INSTALL.md:10]: Typo.

File: INSTALL.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+  - [zgen](#zgen)
+  - [zplus](#zplug)
```
Comment (danielshahaf) [inline INSTALL.md:12]: Typo.

**Review by danielshahaf — COMMENTED** [review]
Caught two typos, but haven't done a full review.

**Comment by tejasc0** [comment]
i guess zplugin has changed its name to zinit, please check this repo and let me know shall i change link to this repo [zinit](https://github.com/zdharma/zinit)

**Comment by phy1729** [comment]
Github already automatically generates a table of contents (although it's not very discoverable). https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

**Comment by tejasc0** [comment]
> Github already automatically generates a table of contents (although it's not very discoverable). https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

then what should we do, shall i close the pull request.

**Comment by phy1729** [comment]
I think in light of Github generating a table of contents we shouldn't manually maintain our own.

**Pull request closed without merging**